docs-ref-conceptual/release-notes-azure-cli.md

```diff
@@ -4,18 +4,114 @@ description: Learn about the latest updates to Azure CLI 2.0
+
+### ACS
+
+* Fixed `get-credentials` command
+* Removed SPN role requirement
+
+### Appservice
+
+* Fixed bug with `config ssl upload` where `hosting_environment_profile` was null
+* Added support for custom URLs to `browse`
+* Fixed slot support for `log tail`
+
+### Backup
+
+* Changed `--container-name` option of `backup item list` to be optional
+* Added storage account options to `backup restore restore-disks`
+* Fixed location check in `backup protection enable-for-vm` to be case insensitive
+* Fixed issue where commands failed with an invalid container name
+* Changed `backup item list` to include 'Health Status' by default
+
+### Batch
+
+* Changed `batch login` to return authentication details
+
+### Cloud
+
+* Changed to not require endpoints when setting `--profile` on a cloud
+
+### Consumption
+
+* Added new commands for reservations [X]
+
+### Event Grid
+
+* [BREAKING CHANGE] Moved the `eventgrid topic event-subscription` commands to `eventgrid event-subscription`
+* [BREAKING CHANGE] Moved the `az eventgrid resource event-subscription` commands to `eventgrid event-subscription`
+* [BREAKING CHANGE] Removed the `eventgrid event-subscription show-endpoint-url` command. Use `eventgrid event-subscription show --include-full-endpoint-url` instead
+* Added command `eventgrid topic update`
+* Added command `eventgrid event-subscription update`
+* Added `--ids` parameter for `eventgrid topic` commands
+* Added tab completion support for topic names
+
+### Interactive
+
+* Fixed issue where interactive mode did not work with Python 2.x
+* Fixed errors on startup
+* Fixed issue with some commands not running in interactive mode
+
+### IoT
+
+* Added support for device provisioning service
+* Added deprecation messages in commands and command help
+* Added IoT check to inform users of the IoT Extension
+
+### Monitor
+
+* Added multi-diagnostic setting support. The `--name` parameter is now required for `monitor diagnostic-settings create`
+* Added command to get diagnostic settings category [X]
+
+### Network
+
+* Fixed issue when trying to change to/from active-standby mode with `vnet-gateway update`
+* Added support for HTTP2 to `application-gateway [create|update]`
+
+### Profile
+
+* Added support for login with user assigned identities
+
+### Role
+
+* Added `--assignee-object-id` argument [X]
+
+### Service Fabric
+
+* Added detailed errors to validation response when creating cluster
+* Fixed missing client issue with several commands
+
+### VM
+
+* [PREVIEW] Cross-zone support for `vmss`
+* [BREAKING CHANGE] Changed single-zone `vmss` default to "Standard" load balancer
+* vm/vmss: use right term of "userAssignedIdentity" for EMSI [X]
+* [PREVIEW] Added support for OS disk swap
+* Added support for using VM images from other subscriptions
+* vmss: ensure app-gateway has a name when defaults to it for large scalesets [X]
+* `vm/vmss create`: expose `plan` arguments for using custom images with billing informations [X]
```

Comment:
Requires information on:

* Which argument(s) were added
* Which commands they were added to

Reply:
addressed.